RFCs/guidelines.md

```diff
@@ -1 +1,19 @@
 # Guidelines
+
+## Directory Structure 
+
+The `RFCs` folder should be the root folder for all RFCs. No other folders should be create on the same level as the `RFCs` folder. 
+
+There are currently three category folders under the root folder: `backend`, `frontend`, `tutorials`. The `backend` folder contains the RFCs that are related to the backend. Similarly, the RFCs for the frontend goes into the `frontend` folder. The `tutorials` folder contains the guidelines for the writers and editors of the tutorials. 
+
+The subfolder, topic folder, in each category folder should concern one and only topic. There should not be any loose documents aside from `introduction.md` in each category folder, that is, every document except `introduction.md` in a category folder should be contained in a topic folder. 
+
+Each topic folder may have subfolders. The subfolders should be used to compartmentalize the documents in the topic folder. The structure of the subfolders depends on the author of the topic folder. 
+
+## Document Naming Convention
+
+The name of each document should be in lower case. The name of each document should be the clear, concise and impart the subject of that document. 
```

Comment:
I'd like to add some examples here, for instance, `something.md` , what do you think?

and how to maintain the multiple words of the single file name, say `some_beautiful_girls.md`.

Reply:
Great suggestions. I'll add that part.

Reply:
If the file name contains spaces, I'd say just keep the spaces. That is, just write it as normal English except no caps.